main.go

```diff
@@ -42,21 +43,22 @@ func main() {
 	stop := make(chan struct{})
 	defer close(stop)
 
-	// TODO: Allow user define multiple types of controllers.
-	switch config.ControllerType {
-	case "endpoint":
-		podController := pod.NewPodController(k8sClientset, calicoClient)
-		go podController.Run(config.EndpointWorkers, config.ReconcilerPeriod, stop)
-	case "profile":
-		namespaceController := namespace.NewNamespaceController(k8sClientset, calicoClient)
-		go namespaceController.Run(config.ProfileWorkers, config.ReconcilerPeriod, stop)
-	case "policy":
-		policyController := networkpolicy.NewPolicyController(k8sClientset, calicoClient)
-		go policyController.Run(config.PolicyWorkers, config.ReconcilerPeriod, stop)
-	default:
-		log.Fatal("Not a valid CONTROLLER_TYPE. Valid values are endpoint, profile, policy.")
+	for _, controllerType := range strings.Split(config.ControllerType, ",") {
```

Comment:
I think we need to change the name of the config option since it's no longer a single type.

How about we rename to `config.EnabledControllers` (env var `ENABLED_CONTROLLERS`)